docs/product-ibm-websphere-installation-and-configuration-guide/src/main/asciidoc/download-extract-ds-proc.adoc

```diff
@@ -0,0 +1,52 @@
+[id='download-extract']
+= Downloading and extracting {KIE_SERVER}
+
+You must download and extract {KIE_SERVER} prior to installing it on WAS.
+
+== Downloading {KIE_SERVER}
+
+Download the deployable {KIE_SERVER} package file for WAS is from the Red Hat Customer Portal.
+
+. Log in to the https://access.redhat.com[Red Hat Customer Portal].
+. Click *DOWNLOADS* at the top of the page.
+. On the *Product Downloads* page that opens, navigate to the *JBOSS INTEGRATION AND AUTOMATION* section, and click *{PRODUCT}*.
+. On the *Software Downloads* page, if necessary select *BRMS* from the Product menu and *7.0* from the *Version* menu.
+. Download the following product distributions:
+* *{PRODUCT} {KIE_SERVER} for All Supported EE7 Containers* (`rhdm-7.0.0.GA-kie-server-ee7.zip`)
+* *{PRODUCT} 7.0.0 Add Ons* (`rhdm-7.0.0.GA-add-ons.zip`)
+. Extract these files to a temporary directory. For example, `__TEMP_DIR__`.
+
+== Extracting {KIE_SERVER}
+
+
+
+ifdef::BA[]
+The downloaded installation ZIP file for {PRODUCT} (`jboss-bpmsuite-{PRODUCT_VERSION}.0.GA-deployable-was9.zip`) contains the {PRODUCT} WAR deployable archive (`business-central.war`), the {KIE_SERVER} WAR deployable archive (`kie-server.war`), and the Dashbuilder WAR deployable archive (`dashbuilder.war`).
+endif::BA[]
+
+ifdef::DM[]
+The downloaded installation ZIP file for {KIE_SERVER} (`rhdm-7.0.0.GA-kie-server-ee7.zip`) contains the {PRODUCT} WAR deployable archive (`kie-server.war`).
+endif::DM[]
+
+From the terminal, extract the downloaded ZIP file to gain access the deployable WAR files:
+
+ifdef::BA[]
+[source]
+----
+unzip rhdm-7.0.0.GA-kie-server-ee7.zip -d `__TEMP_DIR__`
```

Comment:
When you put this in [source] the italic marks are not processed.  I think if you just use ` ` instead of source, it might render correctly (_TEMP_DIR_